src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/DistributedCacheTagHelperFormatter.cs

```diff
@@ -14,18 +14,13 @@ namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
     /// </summary>
     public class DistributedCacheTagHelperFormatter : IDistributedCacheTagHelperFormatter
     {
-        
+
         /// <inheritdoc />
         public Task<byte[]> SerializeAsync(DistributedCacheTagHelperFormattingContext context)
         {
-            if (context == null) 
-            {
-                throw new ArgumentNullException(nameof(context));    
-            }
-            
-            if (context.Html == null) 
```

Comment:
`context.Html == null` is still an error case. Will now `throw` a useless NRE.

Instead fix the `Exception` to be an `ArgumentException` and use our usual `FormatPropertyOfTypeCannotBeNull` resource.